_includes/live-stream-embed.html

```diff
@@ -30,5 +30,5 @@ <h3 id="loadingTitle">Locating stream...</h3>
   </div>
 </div>
 
-<script src="/js/vendor/video.min.js"></script>
-<script src="/js/live-stream-embed.js"></script>
+<script src="/js/vendor/video.min.js?{{site.time | date: '%s%N'}}"></script>
+<script src="/js/live-stream-embed.js?{{site.time | date: '%s%N'}}"></script>
```

Comment:
What is the use of this?

Reply:
This is to get around possible caching issues. We have the same method going for the other instances of JS/CSS on the site.